src/UiPath.Workflow.Runtime/ParallelTrackingExtensions.cs

```diff
@@ -1,5 +1,32 @@
 ﻿namespace System.Activities.ParallelTracking;
 
+[DataContract]

```

Comment:
We discussed that adding this type will cause some backward compatibility issues. When suspending a workflow with a new robot and the resuming it on an old Robot. This scenario needs to be mitigated in some way, either by enforcing it not being supported or, have you found another way?

Reply:
@vuplea was saying that creating a new project with newer packages and running it on older robot is not supported scenario